### DISCUSSION TITLE
Advantage 360 pro broken after updating the firmware to v3

### DISCUSSION THREAD

**Discussion by VinozzZ** [manna-harbour/miryoku#266] [Q&A] [answered]
The advantage 360 had a major update to their firmware. Since then I haven't be able to make it work with the keymap. Could someone help me to figure out how to update the keymap config to make it work again?
Here's their doc for the update: https://github.com/VinozzZ/Adv360-Pro-ZMK#upgrading-from-v2-to-v3

**Comment by manna-harbour** [comment, marked as answer]
Miryoku ZMK supports two different keyboard definitions for the Kinesis Advantage 360 Pro: `adv360` (requires a ZMK fork), and `adv360pro` (can be built from ZMK main).

You can see the repos used for each at https://github.com/manna-harbour/miryoku_zmk/blob/master/.github/workflows/outboards/boards/adv360 and https://github.com/manna-harbour/miryoku_zmk/blob/master/.github/workflows/outboards/boards/adv360pro. 

For `adv360`, Kinesis has removed the v2 branch, so the outboard needed to be updated. That has been done now.

Here are sample build runs:

- `adv360`: https://github.com/manna-harbour/miryoku_zmk/actions/runs/8965407385
- `adv360pro`: https://github.com/manna-harbour/miryoku_zmk/actions/runs/8965250469
